packages/core/src/Stores/client-store.js

```diff
@@ -736,6 +736,52 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    @action.bound
+    redirectToMt5Real = (market_type, server, doRedirect) => {
+        const synthetic_server_region = server ? server.server_info?.geolocation.region : '';
```

Comment:
can we add optional chaining( ? ) to geolocation also?

Reply:
Thanks for advice, updated here.